acceptance/analyzer_test.go

```diff
@@ -27,36 +27,28 @@ import (
 	h "github.com/buildpacks/lifecycle/testhelpers"
 )
 
-const (
-	readWriteImage = "image-readable-writable"
-	onlyReadImage  = "image-readable"
-	onlyWriteImage = "image-writable"
-	noAccessImage  = "noAccessImage"
-)
-
 var (
 	analyzerBinaryDir    = filepath.Join("testdata", "analyzer", "analyze-image", "container", "cnb", "lifecycle")
 	analyzeDockerContext = filepath.Join("testdata", "analyzer", "analyze-image")
 	analyzeImage         = "lifecycle/acceptance/analyzer"
 	analyzerPath         = "/cnb/lifecycle/analyzer"
 	cacheFixtureDir      = filepath.Join("testdata", "analyzer", "cache-dir")
 	daemonOS, daemonArch string
-	readOnlyRegistry     *ih.DockerRegistry
-	authRegistry         *ih.DockerRegistry
-	customRegistry       *ih.DockerRegistry
+	testRegistry         *ih.DockerRegistry
 	registryNetwork      string
 	fixtures             analyzeFixtures
 )
 
 type analyzeFixtures struct {
-	authRegAppImage       string
+	authRegAppImage       string // read and write access
```

Comment:
Maybe it's better to use the convention like in lines 49-51 and call this variable `readWriteAuthRegAppImage`.

Reply:
Same comment apply to lines 45-46.

Reply:
I think we could remove the `AuthReg` (and `readOnlyReg`) part since now we have only one registry.

Reply:
So `readWriteAppImage` and `readOnlyAppImage`, etc.

Reply:
I did a refactoring following the advice, check the following [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/59c111c24edd1c3bb2cdcaba6ba521a5b3718d4d)